### ISSUE TITLE
Support sqlite for db_rebuild_cache

### ISSUE THREAD

**Issue by kpcyrd** [rapid7/metasploit-framework#9076]
It seems the search cache currently requires postgres. It would be nice to have sqlite support for casual use. I've searched in the issue track but it seems this feature wasn't suggested before.

### Metasploit version
4.16.10-dev

### I installed Metasploit with:
- [X] archlinux package via pacman
- [ ] Omnibus installer (nightly)
- [ ] Commercial/Community installer (from http://www.rapid7.com/products/metasploit/download.jsp)
- [ ] Source install (please specify ruby version)

**Comment by jmartin-r7** [comment]
Currently there is a technical restriction that binds metasploit framework to postgres.  A previous decision to model IP addresses using the postgres ip datatype binds storage for the time being.

The current roadmap does include work in progress to separate the data model from framework.  Once this work is complete it may be possible to provide an sqlite backend for storage.

https://github.com/rapid7/metasploit-framework/wiki/2017-Roadmap

**Comment by adfoster-r7** [comment]
Closing, as `db_rebuild_cache` has been removed
